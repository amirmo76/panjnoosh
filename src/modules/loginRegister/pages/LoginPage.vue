<template>
  <div class="container">
    <form>
      <BaseInput :type="'email'" @valueChange="onEmailChange">ایمیل</BaseInput>
      <BaseInput :type="'password'" @valueChange="onPasswordChange">رمز عبور</BaseInput>
      <BaseButton :stretched="true">ورود</BaseButton>
    </form>
  </div>
</template>

<script>
import BaseInput from "../../../core/components/BaseInput";
import BaseButton from "../../../core/components/BaseButton";

export default {
  name: "LoginPage",
  data: {
    email: "",
    password: "",
    emailError: "",
    passwordError: ""
  },
  components: {
    BaseInput,
    BaseButton
  },
  methods: {
    onEmailChange: () => {},
    onPasswordChange: () => {}
  }
};
</script>

<style scoped>
.container {
  background: #1d1d1d;
  /* min-height: 100vh; */
  display: flex;
  flex-direction: column;
  align-items: center;
}

form {
  display: flex;
  flex-direction: column;
  align-items: center;
}

form > *:not(:last-child) {
  margin-bottom: 4rem;
}
</style>


