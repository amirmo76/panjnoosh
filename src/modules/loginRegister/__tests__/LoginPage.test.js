import { shallowMount } from "@vue/test-utils";
import LoginPage from "../pages/LoginPage";

test("should match snapshot", () => {
  const wrapper = shallowMount(LoginPage);
  expect(wrapper.element).toMatchSnapshot();
});
