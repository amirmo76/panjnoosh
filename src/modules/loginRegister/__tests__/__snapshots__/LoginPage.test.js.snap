// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  class="page-container"
>
  <div
    class="image"
  />
   
  <div
    class="left-box-container"
  >
    <h1>
      ورود | پنج نوش
    </h1>
     
    <div
      class="form-and-text-container"
    >
      <loginform-stub />
       
      <div
        class="text"
      >
        <h3>
          خوش آمدید
        </h3>
         
        <p>
          لورم ایپسوم متن ساختگی با تولید سادگی نامفهوم از صنعت چاپ و با استفاده از طراحان گرافیک است. چاپگرها و متون بلکه روزنامه و مجله در ستون و سطرآنچنان که لازم است و برای شرایط فعلی تکنولوژی مورد نیاز و کاربردهای متنوع با هدف بهبود ابزارهای کاربردی می باشد.
        </p>
         
        <h3>
          توضیحاتی کوتاه
        </h3>
         
        <p>
          لورم ایپسوم متن ساختگی با تولید سادگی نامفهوم از صنعت چاپ و با استفاده از طراحان گرافیک است. چاپگرها و متون بلکه روزنامه و مجله در ستون و سطرآنچنان که لازم است و برای شرایط فعلی تکنولوژی مورد نیاز و کاربردهای متنوع با هدف بهبود ابزارهای کاربردی می باشد.
        </p>
      </div>
    </div>
  </div>
</div>
`;
